README.md

```diff
@@ -1,46 +1,41 @@
-- `messages` - crate encapsulating Aries message models and builders
-- `diddoc` - crate for working with DIDDocs
-- `agency-client` - client to communicate with [vcx mediator](https://github.com/AbsaOSS/vcxagencynode)   
+This is repository of
+- Rust library `aries-vcx` implementing Aries protocols
+- and collection of supporting projects
 
-## Additional crates:
-Additionally, you can find here project built on **top of `aries-vcx`**:
-- `agents/rust/aries-vcx-agent` - simple agent implementation in rust on top of `aries-vcx` crate
-- [`libvcx`](libvcx) - built on top of `aries-vcx`, is a particular approach how to use `aries-vcx` on
-  mobile or from other languages.
+## If you are Rust 🦀 developer
+You can build your Rust project on top of
+- [`aries-vcx`](aries_vcx) - Ready to go Rust library to work with Aries protocols, both from
+issuer/verifier side or credential holder/prover.
 
-## Getting started
-- Ask question on [discord](https://discord.com/channels/905194001349627914/955480822675308604)
-- Talk to us on community call starting every Thursday 09:00am UTC via [zoom](https://zoom.us/j/97759680284?pwd=VytRRlJSd3c5NXJ1V25XbUxNU0Jndz09)
-- See high level 2023 roadmap at [ROADMAP_2023.md](ROADMAP_2023.md)
-- Find out what's planned in [issues](https://github.com/hyperledger/aries-vcx/issues) 
-  and project [board](https://github.com/orgs/hyperledger/projects/14)
-- We welcome new contributors! Connect with us via the channels above and take a look at [CONTRIBUTING.md](CONTRIBUTING.md)
-
-## Versioning
-- We are currently not following semantic versioning. Version are releasing `0.x.x` versions. 
-- Breaking changes to APIs happen occasionally. See full changelogs records at 
-  [releases](https://github.com/hyperledger/aries-vcx/releases) page.
+Additionally, Aries-vcx implementation is built on top of smaller Rust crates:
+- [`messages`](messages) - crate for building and parsing Aries messages
+- [`diddoc`](diddoc) - crate to work with DIDDocs
 
-## CI artifacts
-Following artifacts are build with every CI run and release:
+## If you are mobile 📱 developer
+Aries-vcx is well positioned for native Rust applications, however we are currently transitioning 
+the recommended approach:
+- deprecated, but battle-tested: `libvcx` and its Java and ObjectiveC wrappers is battle-tested but now deprecated approach.
+- encouraged, but in stage of POC: `uniffi_vcx` is next generation approach to enable `aries-vcx` on mobile, providing Swift
+and Kotlin wrappers. However, this is yet in POC stage and new adopters `aries-vcx` for mobile
+are highly encourage to contribute to its development. 
 
-### Github Actions artifacts
-  - *(these are to be found at bottom of Summary page for each CI run)*
-  - `libvcx.so`, `libvcx.dylib` - dynamic library for x86_64 ubuntu, x86_64 darwin)
-  - ios and java wrapper built on top of `libvcx`
+Read more info about the transition and deprecation.
```

Comment:
Is this meant to be a link? I.e. to the libvcx readme deprecation notice

Reply:
good catch!